radixdlt-java/src/main/java/com/radixdlt/client/application/RadixApplicationAPI.java

```diff
@@ -176,12 +204,31 @@ public static RadixApplicationAPI create(
 		RadixUniverse universe,
 		FeeMapper feeMapper
 	) {
-		Objects.requireNonNull(identity);
-		Objects.requireNonNull(universe);
-		Objects.requireNonNull(feeMapper);
 		return new RadixApplicationAPI(identity, universe, feeMapper, universe.getLedger());
 	}
 
+	/**
+	 * For advanced use at the moment.
+	 *
+	 * Create an api based on actionToParticleMappers of own choosing.
+	 *
+	 * @param identity identity for api
+	 * @param actionToParticlesMappers the mappers to utilize for api
+	 * @return api object used to interact with ledger
+	 */
```

Comment:
Nitpicking2: A beautifully written JavaDoc method description uses a 3rd person declarative form ("Gets this and that") rather than 2nd person imperative form ("Get this and that").

If you, just as I, get excited about JavaDoc recommendations, there is a good article about it here: https://www.oracle.com/technetwork/java/javase/index-137868.html

Reply:
Unfortunately, I've always been lackadaisical with javadocs which is why you're here! Cool resource and will hopefully improve on this with your guidance :)

Reply:
You’re so humble! You don’t deserve an ass hole reviewer like me...